sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Snippets/Sample05_IdentityAndSharedAccessCredentialsLiveTests.cs

```diff
@@ -135,7 +135,7 @@ public async Task SharedAccessSignature()
             /*@@*/ storageConnectionString = StorageTestEnvironment.Instance.StorageConnectionString;
             /*@@*/ blobContainerName = storageScope.ContainerName;
 
-            //@@ var credential = new AzureSasCredential("<< SHARED ACCESS KEY STRING >>");
+            var credential = new AzureSasCredential("<< SHARED ACCESS KEY STRING >>");
```

Comment:
Orthogonal, but I forgot the SAS credential doesn't do eager validation; we can use this form.